### DISCUSSION TITLE
How do I get Video format other than GIF?

### DISCUSSION THREAD

**Discussion by andradei** [SeaDve/Kooha#295] [Q&A] [answered]
I'm on Arch, KDE Plasma 6 `xdg-desktop-portal` and `xdg-desktop-portal-kde`.

The only Video format I can choose is GIF.
How do I get a format with video and mic recording?

**Comment by SeaDve** [comment, marked as answer]
I assume it is not installed via Flatpak; you have to have the gst-plugins-* installed.
